api/v1alpha6/openstackmachinetemplate_webhook_test.go

```diff
@@ -150,6 +150,7 @@ func TestOpenStackMachineTemplate_ValidateUpdate(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
I don't understand why this showed up now. Make lint was complaining, but for me it was complaining also on `main`, but I don't see it complaining on other PRs. :shrug:

Reply:
um... this is weird .. but should not block our merge for this PR?

Reply:
Yes it is fine now I was just unsure why we didn't see it before when running `make lint`. Anyway it is fixed in this PR. :slightly_smiling_face: